### DISCUSSION TITLE
npm 怎么安装青龙

### DISCUSSION THREAD

**Discussion by xiezy1** [whyour/qinglong#2076] [Q&A]
我是用的win10 已经安装了node/npm/python/pip/git
![image](https://github.com/whyour/qinglong/assets/63857131/10a2ddc0-1e31-47ab-bcad-8fb38f202b6a)

**Comment by xiezy1** [comment]
![image](https://github.com/whyour/qinglong/assets/63857131/4ffea2f9-b698-493e-bcc5-dee4868956a7)

**Reply by whyour** [reply]
npm i -g node-pre-gyp 然后再重新装试试
发下你的系统版本

**Comment by dmh91168** [comment]
目前没有本地化版本，都是在容器中运行的
